fairscale/__init__.py

```diff
@@ -3,10 +3,6 @@
 # This source code is licensed under the BSD license found in the
 # LICENSE file in the root directory of this source tree.
 
-# Please update the doc version in docs/source/conf.py as well.
-__version_tuple__ = (0, 4, 3)
-__version__ = ".".join([str(x) for x in __version_tuple__])
```

Comment:
I think __version__ here is to provide __version__ attribute for developers to use at run time. 
https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package

Reply:
I propose assign a value to __version__  similar way as the set up file.

Reply:
good catch. fixed it.

Reply:
We need to update the value in the `__version_tuple__`. The cleanest way to do it is to move it into a separate file `version.py` containing only the `__version_tuple__` variable and update it when needed.